tests/common/test_models_builder.py

```diff
@@ -12,38 +12,44 @@ def test_documentbuilder():
 
     # Don't resolve lines
     doc_builder = builder.DocumentBuilder(resolve_lines=False, resolve_blocks=False)
-    boxes = np.random.rand(words_per_page, 6)
-    boxes[:2] *= boxes[2:4]
+    boxes = {"words": np.random.rand(words_per_page, 6)}
```

Comment:
Let's add another test to check if "old" format is still working with this code

Reply:
Up

Reply:
Done